article/app/controllers/ArticleController.scala

```diff
@@ -160,3 +175,15 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
   }
 
 }
+
+object LongCacheAction {
+  def apply[X](block: RequestHeader => Future[Result]) = {
```

Comment:
\<pedant\>It's a type parameter, which in this case isn't covariant, that would be `[+X]`\</pedant\>

I agree it seems redundant though.